test/Dialect/LLHD/IR/arithmetic.mlir

```diff
@@ -1,12 +1,29 @@
-//RUN: circt-opt %s | circt-opt | FileCheck %s
+// RUN: circt-opt %s -mlir-print-op-generic | circt-opt | circt-opt | FileCheck %s
 
-// CHECK: func @check_arithmetic(%[[A:.*]]: i64, %[[B:.*]]: i64) {
-func @check_arithmetic(%a : i64, %b : i64) -> () {
+// CHECK-LABEL: @check_arithmetic
+// CHECK-SAME: %[[A:.*]]: i64
+// CHECK-SAME: %[[VEC:.*]]: vector<3xi32>
+// CHECK-SAME: %[[TUP:.*]]: tuple<i1, i2, i3>
+func @check_arithmetic(%a : i64, %vec : vector<3xi32>, %tup : tuple<i1, i2, i3>) -> () {
     // CHECK-NEXT: %{{.*}} = llhd.neg %[[A]] : i64
```

Comment:
Might be worth to have 2-space indents here too to keep it consistent.

Reply:
Yes, we should change all mlir files to 2-space indent for consistency, but currently all mlir files in LLHD have 4-space indent, so I think it might be better to change the existing ones in one go in a separate commit.